loris/resolver.py

```diff
@@ -61,13 +65,40 @@ def resolve(self, ident):
             ident (str):
                 The identifier for the image.
         Returns:
-            (str, str): (fp, format)
+            ImageInfo: Partially constructed ImageInfo object 
```

Comment:
Wouldn't it be better to construct the ImageInfo object in one step, instead of having a partially constructed object? We could pass the transformers from webapp.py into resolve(), which would let us do the whole thing in one step. Or, we could have resolve() not return the ImgInfo object, but just return the additional path to the rules file (if it exists), and then the ImgInfo object could be constructed outside of the resolvers. Thoughts?